SilasKenneth/fast-food-fast-ui
# [Finishes #160131579] Create a user login page

#### What this pull request does
Creates the page where users are able to login
##### Description of the task to be completed
 Create a page where a user can log into their existing accounts
    -  Add the form for the login
    - Add links for the user to choose other options
##### Relevant pivotal tracker story
[#160131579](https://www.pivotaltracker.com/n/projects/2194200/stories/160131579)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder, ```git checkout the ft-login-160131579```
  - Open the login.html file in your browser and that should work

#### Screenshot
![Login page](https://user-images.githubusercontent.com/11555354/44915763-7f4a0300-ad3c-11e8-9113-6a126e87f2a0.png)
